tests/utils/utils.py

```diff
@@ -0,0 +1,33 @@
+from ape import chain
+
+
+def vault_status(vault):
+    print(f"--- Vault {vault.name()} ---")
+    print(f"API: {vault.apiVersion()}")
+    print(f"TotalAssets: {to_units(vault, vault.totalAssets())}")
+    print(f"PricePerShare: {to_units(vault, vault.pricePerShare())}")
+    print(f"TotalSupply: {to_units(vault, vault.totalSupply())}")
+
+
+def strategy_status(vault, strategy):
+    status = vault.strategies(strategy).dict()
+    print(f"--- Strategy {strategy.name()} ---")
```

Comment:
Please do not do test that output values, they aren't check by CI make assertion instead